bin/tape

```diff
@@ -26,10 +26,13 @@ opts.require.forEach(function(module) {
 });
 
 opts._.forEach(function (arg) {
-    glob(arg, function (err, files) {
-        files.forEach(function (file) {
-            require(resolvePath(cwd, file));
-        });
+    var files;
+    try {
+        files = glob.sync(arg);
```

Comment:
When in the async version would an error be generated by glob, compared to the sync version?

Separately, I appreciate that the current PR mirrors the original version's error handling precisely; but it might be better to let `glob.sync`'s error throw, rather than having an `undefined.forEach` error. I'd consider that change semver-patch, since it's just making the error message more helpful.